weatherAPI/models/object.go

```diff
@@ -0,0 +1,53 @@
+package models
```

Comment:
all this stuff can be deleted, is part of the boilerplate.

Reply:
Deleted